styles.css

```diff
@@ -111,9 +118,432 @@ nav {
 ul {
   display: flex;
   flex-direction: row;
+  margin-right: 30px;
+  flex-wrap: wrap;
+  justify-content: center;
+  list-style-type: none;
+  text-align: center;
+}
+
+.drop-down {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  margin: auto;
+}
+
+.works {
+  max-width: 327px;
+  height: 97px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+body {
+  max-width: 375px;
+  align-items: center;
+  justify-content: center;
   margin: auto;
+}
+
+.indicator {
   width: 100%;
-  flex-wrap: wrap;
+}
+
+.block {
+  width: 327px;
+  height: 474px;
+  top: 4px;
+  left: 2px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  margin-bottom: 24px;
+}
+
+.gray-area {
+  height: 252px;
+  width: 327px;
+  background: #ebf0ee;
+}
+
+.rest {
+  margin: auto;
+  background-color: #fff;
+  padding-left: 25px;
+}
+
+#line {
+  margin: auto;
+  align-items: center;
   justify-content: center;
+  display: flex;
+  padding-right: 25px;
+}
+
+.description {
+  background: #fff;
+  border-radius: 8px;
+  border: 2px solid #ebf0ee;
+  height: 222px;
+}
+
+.post {
+  width: 295px;
+  height: 57px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  margin: 16px auto;
+  flex-grow: 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 305px;
+  height: 32px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: auto;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix the style error highlighted in the image below!

![image](https://user-images.githubusercontent.com/63915024/187052860-8ff5f900-b6d4-4e6b-af4e-b4d11fb3d9f2.png)